config/notifiers.go

```diff
@@ -306,6 +317,37 @@ func (c *OpsGenieConfig) UnmarshalYAML(unmarshal func(interface{}) error) error
 	return checkOverflow(c.XXX, "opsgenie config")
 }
 
+// VictorOpsConfig configures notifications via VictorOps.
+type VictorOpsConfig struct {
+	NotifierConfig `yaml:",inline"`
+
+	APIKey       Secret `yaml:"api_key"`
+	APIURL       string `yaml:"api_url"`
+	RoutingKey   string `yaml:"routing_key"`
+	MessageType  string `yaml:"message_type"`
+	EntityID     string `yaml:"entity_id"`
```

Comment:
Is this visible to the end user in VictorOps? If so we probably want something prettier as is currently done.